README.es-ES.md

```
@@ -0,0 +1,80 @@
+  - Velocidad máxima del ventilador
+  - Velocidad mínima del ventilador
+  - Velocidad actual del ventilador
+  - Velocidad objetivo del ventilador
+- Expone métricas de temperatura de numerosos sensores
+
+
+
+## Ejecución del exportador
+
+1. Clona el repositorio:
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)